Per Ashrit Shetty: Update readme for error message (#637)
microsoft/DirectML: Samples/DirectML_ESRGAN/README.md

You must have Windows 11 24H2 installed to run this sample using an NPU.

<sup>1</sup> Currently, this sample is only supported with the NPU available in Copilot+ PCs with Snapdragon X Series processors. Support for additional NPUs will come soon. To run this sample with the QC NPU you must install driver 30.0.31.250 or newer.

> <sup>2</sup> The following message is expected and trust that the model has executed successfully. The "Error in cpuinfo" and "Unknown Qualcomm CPU" messaging you received is a print message done in error. It will be removed in an upcoming release.

Reviewer: Propose rewording to start with the error message first:

```suggestion
<sup>2</sup> The following messages "Error in cpuinfo" and "Unknown Qualcomm CPU" are mistaken messages which will be fixed in a future release. They are expected, and the model has executed successfully.
```
